src/kernelsMod.f90

```diff
@@ -122,13 +141,13 @@ subroutine pathlength_scatter(input_file)
             if(id == 0 .and. mod(j,1000) == 0)then
                 if(state%tev)then
 !$omp critical
-                    image = reshape(jmean(:,100:100,:), [state%grid%nxg,state%grid%nzg,1])
```

Comment:
Please change these back to jmean for the time being. Though it would be useful to see the phase in tev. Though this will need some careful thought before implmentation